libs/components/src/lib/selectable-box/selectable-box.ts

```diff
@@ -98,8 +82,9 @@ export class SelectableBox extends FoundationElement {
 	 * @internal
 	 */
 	_handleKeydown(event: KeyboardEvent) {
+		if (!this.clickable) return true;
 		if (event.code === 'Space' || event.code === 'Enter' && this.clickable) 
```

Comment:
I guess is should've been like this
```suggestion
		if ((event.code === 'Space' || event.code === 'Enter') && this.clickable) 
```